_includes/education.html

```diff
@@ -0,0 +1,19 @@
+<h2>Education</h2>
```

Comment:
Same comment as before. Education section is extra and not needed for a pod portfolio. This was not mentioned in the issue either.